net/bulk_emulation/mem_xprt_pvt.h

```diff
@@ -36,9 +36,9 @@
 */
 
 enum {
-	M0_NET_BULK_MEM_MAX_BUFFER_SIZE     = (1 << 19),
-	M0_NET_BULK_MEM_MAX_SEGMENT_SIZE    = (1 << 18),
-	M0_NET_BULK_MEM_MAX_BUFFER_SEGMENTS = 256,
+	M0_NET_BULK_MEM_MAX_BUFFER_SIZE     = (1 << 20),
+	M0_NET_BULK_MEM_MAX_SEGMENT_SIZE    = (1 << 20),
+	M0_NET_BULK_MEM_MAX_BUFFER_SEGMENTS = 1,
```

Comment:
I thought I had asked these questions.

Why are these constants for "mem bulk" changed?
If we need some modification to libfabric, why is this related?

Reply:
In the main branch, with lnet, the rpc bufferpool and ios-bufferpool is initialized with config seg_size=4k seg_nr=256 buf_size=1Mb which is derived from the rpc_seg_size, rpc_seg_nr.
But now we can have different configs for rpc buffer pool and bulk buffer pool.
To initialize the bulk(ios) bufferpool, the max_buf_size = max_seg_nr x max_seg_size does not match for the bulk mem xprt. (max_buf_size=512k max_seg_nr=256 max_seg_size=256k)
Hence we changed these values to max_buf_size = 1MB max_seg_nr=1 max_seg_size=1MB
However, we can have some different config as long as it fulfills this criteria
(max_buf_size = max_seg_nr x max_seg_size)
 
So should we change the config to max_buf_size = 1MB max_seg_nr=256 max_seg_size=4k ? (same as what we have in lnet/libfabric at present)?